language/Russian/strings.po

```diff
@@ -6465,12 +6484,12 @@ msgstr "Команда"
 #: plugins/data/data_story.lua:24
 msgctxt "plugin-data"
 msgid "Mamescore"
-msgstr ""
+msgstr "Рекорд"
 
 #: plugins/data/data_sysinfo.lua:16
 msgctxt "plugin-data"
 msgid "Sysinfo"
-msgstr ""
+msgstr "Информация о системе"
```

Comment:
I’m not sure these should be translated literally – they refer to specific data files that were at one time maintained by community members.